examples/ws/soap/StoreExample/StoreServer/src/MyStoreServer.java

```diff
@@ -0,0 +1,43 @@
+import java.util.HashMap;
+
+public class MyStoreServer {
+	
+	HashMap<String, Integer> inventory = new HashMap<>();
```

Comment:
Naming is off again, but I guess it comes from the sample?

Reply:
it is from the sample, will change it.